docs/cluster-management/config.md

```diff
@@ -65,6 +65,9 @@ operator_load_balancer_scheme: internet-facing  # must be "internet-facing" or "
 # CloudWatch log group for cortex (default: <cluster_name>)
 log_group: cortex
 
+# tags are assigned to aws resources for labelling and cost allocation
+tags: # cortex.dev/cluster-name: <cluster_name>
```

Comment:
Did you confirm that the config can be read without error if `tags` is left blank?